powersimdata/scenario/ready.py

```diff
@@ -0,0 +1,127 @@
+import copy
+
+from powersimdata.input.input_data import get_bus_demand
+from powersimdata.input.transform_profile import TransformProfile
+from powersimdata.scenario.state import State
+
+
+class Ready(State):
+    exported_methods = {
+        "print_scenario_info",
+        "get_ct",
+        "get_grid",
+        "get_bus_demand",
+        "get_demand",
+        "get_hydro",
+        "get_grid",
+        "get_solar",
+        "get_wind",
+        "get_wind_offshore",
+    }
+
+    def __init__(self, scenario):
+        """Constructor."""
+        super().__init__(scenario)
+
+    def _update_scenario_info(self):
+        """Updates scenario information."""
+        pass
```

Comment:
How does it work? Where is the update happening?

Reply:
When called from analyze state, we default to this. But execute overrides the method, so it will still download the scenario list.

Reply:
BTW, I'm working on moving this to state.py instead, so it can also be used from the create state, which also defines its own version of `_update_scenario_info`

Reply:
The `Scenario` class has a `print_scenario_info` method. Just wondering if the inheritance is not going to break somewhere.

Reply:
Yeah I was concerned about that. After testing again it seems to work fine. Changed it so `print_scenario_info` is exported from `scenario.state`, so the `Scenario` doesn't have to explicitly proxy.